spec/lib/synced/strategies/updated_since_spec.rb

```diff
@@ -107,6 +107,33 @@
           }.to change { Synced::Timestamp.with_scope_and_model(account, LosRecord).last_synced_at }.from(nil).to(future_sync_time)
         end
       end
+
+      context "with ENV['LAST_SYNCED_AT_OFFSET'] specified" do
+        let(:request_timestamp) { 1.year.ago }
+
+        before do
+          ENV["LAST_SYNCED_AT_OFFSET"] = "-60"
```

Comment:
I think we should take the absolute of this value, we don't want the tolerance to be ever smaller than 0, that would lead to sync issues.